src/test/java/delete_task_use_case/DeleteTaskInteractorTest.java

```diff
@@ -0,0 +1,37 @@
+package delete_task_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessEvent;
+import data_access.DataAccessTask;
+import data_access.WeekDataAccess;
+
+import java.util.HashMap;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class DeleteTaskInteractorTest extends WeekDataAccess {
```

Comment:
Initialize an instance of your use case Data Access class to check if the days attribute is mutated appropriately.

Reply:
Done, as suggested.